### PR TITLE
Issues with Awaiting Merge PR

### PR DESCRIPTION
**Pull request by GalaxyGamingBoy**

### CODE REVIEW

**Review by odysseuskir — CHANGES_REQUESTED** [review]
It's very good, but I have a few small points to fix.
1. We should remove the message for the bug since it has been fixed
2. I think the X emoji is confusing for lessons that have not been completed
3. The title of the lesson is incorrect, and the icon is not loading
image
4. This will probably get fixed when we merge, the C++ course doesn't appear, but I guess it's not an issue, I am just pointing this out.

That's all. Great work, we need a few more lines to cross and it will be perfect. GG @GalaxyGamingBoy!

**Comment by Nikos1508** [comment]
It is really good but I would like some changes. 
1) I think that the buttons on the view page (image 1) should be a little bit taller.

2) We should allow users to mark a lesson as undone if they want to. Because users click on a lesson doesn't mean that they have completed this lesson.

3) I think that in this brunch we should make the change that we talked about in the last SC meeting. (to add a search bar and a drop-down between the title and the 1st lesson (image 2).

image
image

**Review by Nikos1508 — COMMENTED** [review]
It is really good but I would like some changes. 
1) I think that the buttons on the view page (image 1) should be a little bit taller.

2) We should allow users to mark a lesson as undone if they want to. Because users click on a lesson doesn't mean that they have completed this lesson.

3) I think that in this branch we should make the change that we talked about in the last SC meeting. (to add a search bar and a drop-down between the title and the 1st lesson (image 2).

image
image

**Comment by Nikos1508** [comment]
Okay, I will make an issue about it. But it is a really small change so I thought that it could be done by the 25th of May.

**Comment by odysseuskir** [comment]
It's not small

File: src/pages/index.astro [hunk 1]
```diff
@@ -23,26 +22,26 @@ const courses = (await getEntry("lessons", "courses")).data.courses.map(
     </div>
     <br />
     <div
-      class="d-flex justify-content-center"
-      style="color: white; padding-left: 50px; padding-right: 50px"
+        class="d-flex justify-content-center"
+        style="color: white; padding-left: 50px; padding-right: 50px"
     >
         <div
```
Comment (odysseuskir) [inline src/pages/index.astro:28]: Remove this

**Review by odysseuskir — APPROVED** [review]
Perfect!

**Pull request merged**